api/services/userService.js

```diff
@@ -0,0 +1,41 @@
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+
+const { userDao } = require("../models");
+
+const signIn = async (email, password) => {
+  const emailRegex = /^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*/;
+  const passwordRegex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.{8,15})/;
+
+  if (!emailRegex.test(email)) {
+    const error = new Error("INVALID_EMAIL");
+    error.statusCode = 400;
+
+    throw error;
+  }
+
+  if (!passwordRegex.test(password)) {
+    const error = new Error("INVALID_PASSWORD");
+    error.statusCode = 400;
+
+    throw error;
+  }
+
+  const user = await userDao.getUserByEmail(email);
+  const match = await bcrypt.compare(password, user.password);
+
+  if (!match) {
+    const error = new Error("WRONG_PASSWORD");
+    error.statusCode = 401;
+
+    throw error;
+  }
+
+  const accessToken = jwt.sign({ id: user.id }, process.env.JWT_SECRET_KEY);
+
+  return accessToken;
```

Comment:
```suggestion
  return jwt.sign({ id: user.id }, process.env.JWT_SECRET_KEY);
```
변수 선언 없이 곧바로 리턴해주어도 될 것 같습니다-!